diff --git a/src/Android.mk b/src/Android.mk
index b792434..9d71f3e 100644
--- a/src/Android.mk
+++ b/src/Android.mk
@@ -14,7 +14,7 @@ LOCAL_SRC_FILES:= bt-av-io.c \
                   main.c \
                   service.c
 LOCAL_C_INCLUDES := system/libfdio/include
-LOCAL_CFLAGS := -DANDROID_VERSION=$(PLATFORM_SDK_VERSION) -Wall -Werror
+LOCAL_CFLAGS := -DANDROID_VERSION=$(PLATFORM_SDK_VERSION) -Wall
 LOCAL_SHARED_LIBRARIES := libfdio \
                           libhardware \
                           libhardware_legacy \

