From 9cf2faadcb068e42ad85272cdf2a525d12280c13 Mon Sep 17 00:00:00 2001
From: viralwang <vwang@mozilla.com>
Date: Thu, 21 May 2015 15:48:03 +0800
Subject: [PATCH] change NFCSTATUS_INVALID_DEVICE define as -1

---
 halimpl/pn547/common/phNfcStatus.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/halimpl/pn547/common/phNfcStatus.h b/halimpl/pn547/common/phNfcStatus.h
index d84b344..64098cc 100644
--- a/halimpl/pn547/common/phNfcStatus.h
+++ b/halimpl/pn547/common/phNfcStatus.h
@@ -88,7 +88,7 @@
 /*
  * Device specifier/handle value is invalid for the operation
  */
-#define NFCSTATUS_INVALID_DEVICE                              (0x0006)
+#define NFCSTATUS_INVALID_DEVICE                              (-1)
 
 /*
  * The function executed successfully but could have returned
-- 
1.7.10.4

