From 966187c24c2d9b8b2cbf80caca85a75043cf0402 Mon Sep 17 00:00:00 2001
From: Michael Wu <mwu@mozilla.com>
Date: Fri, 26 Jul 2013 15:04:53 -0400
Subject: [PATCH] Make b2g binaries executable

---
 include/private/android_filesystem_config.h |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/private/android_filesystem_config.h b/include/private/android_filesystem_config.h
index 5d9c3ea..ee922d6 100644
--- a/include/private/android_filesystem_config.h
+++ b/include/private/android_filesystem_config.h
@@ -223,6 +223,9 @@ static const struct fs_path_config android_dirs[] = {
 ** and will allow partial matches.
 */
 static const struct fs_path_config android_files[] = {
+    { 00755, AID_ROOT,      AID_ROOT,      0, "system/b2g/b2g" },
+    { 00755, AID_ROOT,      AID_ROOT,      0, "system/b2g/updater" },
+    { 00755, AID_ROOT,      AID_ROOT,      0, "system/b2g/plugin-container" },
     { 00440, AID_ROOT,      AID_SHELL,     0, "system/etc/init.goldfish.rc" },
     { 00550, AID_ROOT,      AID_SHELL,     0, "system/etc/init.goldfish.sh" },
     { 00440, AID_ROOT,      AID_SHELL,     0, "system/etc/init.trout.rc" },
-- 
1.7.10.4

